docs/lib/QUICKSTART.md

```diff
@@ -451,17 +451,18 @@ Once the uVisor debug messages are enabled, your application fails. The failure
                     BUS FAULT
 ***********************************************************
 
-...
+* Active Box ID: 0
+* FAULT SYNDROME REGISTERS
 
-/* MEMORY MAP
-  Address:           0x4004800C
-  Region/Peripheral: SIM
-    Base address:    0x40047000
-    End address:     0x40048060
+  CFSR: 0x00008200
+  BFAR: 0x40048044
```

Comment:
OK. I just want to make sure that if someone is following along with the guide, they see what we wrote here.